diff --git a/src/main/index.ts b/src/main/index.ts
index 5a55f40..3b91c40 100644
--- a/src/main/index.ts
+++ b/src/main/index.ts
@@ -17,10 +17,6 @@ import { registerScreenShareHandler } from "./screenShare";
 import { Settings, State } from "./settings";
 import { isDeckGameMode } from "./utils/steamOS";
 
-if (!IS_DEV) {
-    autoUpdater.checkForUpdatesAndNotify();
-}
-
 console.log("Vesktop v" + app.getVersion());
 
 // Make the Vencord files use our DATA_DIR

